src/pages/en/core-concepts/astro-components.md

```diff
@@ -125,19 +125,56 @@ They can be used to style your components, and all style rules are automatically
 
 ### Client-Side Scripts
 
-To send JavaScript to the browser without [using a framework component](/en/core-concepts/component-hydration) (React, Svelte, Vue, Preact, Solid...) you can use a `<script>` tag in your Astro component body.
+To send JavaScript to the browser without [using a framework component](/en/core-concepts/component-hydration) (React, Svelte, Vue, Preact, SolidJS, AlpineJS, Lit...) you can use a `<script>` tag in your Astro component template and send JavaScript to the browser that executes in the global scope.
```

Comment:
Good call! I haven't gone through to update allll the other pages that previously could only link to component-hydration. That's absolutely a to-do, because we're changing the content of that link into one of either "framework-component" or "partial-hydration."

Reply:
OK, belive I found all the links in the site referencing component-hydration and replaced them with one of the two new files.